exocom-server/bin/build

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
+set -ex
```

Comment:
Do we really need to print the command when running `build`? This command runs only one other command. It should be pretty obvious what command is going wrong in case there is an error.